src/styles/pages/StoreItemPage.module.scss

```diff
@@ -0,0 +1,81 @@
+@use 'src/styles/vars.scss' as vars;
+
+.optionsContainer {
+  display: flex;
+  flex: none;
+  flex-direction: column;
+  gap: 1.1rem;
+  max-width: 22.5rem;
+  min-width: 18rem;
```

Comment:
it still feels like the layout depends on the length of the text

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/5ec37fdf-2f82-4a85-a562-850ae88352b1)

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/2a9ff277-c775-4062-8651-e6b33d2f605e)

adding `width: 100%` here fixes this